packages/datadog-plugin-elasticsearch/test/index.spec.js

```diff
@@ -25,11 +25,11 @@ describe('Plugin', () => {
         })
 
         after(() => {
-          return agent.close()
+          return agent.close({ ritmReset: false })
         })
 
         beforeEach(() => {
-          elasticsearch = require(`../../../versions/${moduleName}@${version}`).get()
+          elasticsearch = proxyquire(`../../../versions/${moduleName}@${version}`, {}).get()
```

Comment:
No need for proxyquire here. It can actually cause some issues in plugin tests in some cases so should definitely be avoided when requiring the underlying module itself.

Reply:
I wonder if removing that would also remove the need for `ritmReset`, definitely something to try.

Reply:
ritmReset is needed